client.go

```diff
@@ -705,6 +705,12 @@ func (c *Client[TTx]) StopAndCancel(ctx context.Context) error {
 	return c.awaitStop(ctx)
 }
 
+// Stopped returns a channel that will be closed when the Client has stopped.
+// It can be used to wait for a graceful shutdown to complete.
```

Comment:
Maybe add a sentence on what happens with it when a stop context is cancelled?

Reply:
Done ✅